contrib/spdlog/template.py

```diff
@@ -0,0 +1,29 @@
+pkgname = "spdlog"
+pkgver = "1.13.0"
+pkgrel = 0
+build_style = "cmake"
+configure_args = [
+    "-DSPDLOG_FMT_EXTERNAL=ON",
+    "-DSPDLOG_BUILD_SHARED=ON",
+    "-DSPDLOG_BUILD_EXAMPLE=ON",
```

Comment:
```suggestion
    "-DSPDLOG_BUILD_EXAMPLE=OFF",
```
building the examples doesn't do anything except compile something that isn't installed